docs/HOWTO.md

```diff
@@ -1,17 +1,34 @@
+# How-To at a glance
+
+<div align="right" markdown="1">
+
 *[Read this in other languages](../README.md#translations)*
 
-Welcome to Free-Programming-Books! We welcome new contributors; even those making their very first pull request on Github. If you're one of those, here are some resources that might help:
+</div>
+
+**Welcome to `Free-Programming-Books`!**
+
+We welcome new contributors; even those making their very first Pull Request (PR) on GitHub. If you're one of those, here are some resources that might help:
+
+* [About Pull Requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
+* [Creating a Pull Requests](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)
+* [GitHub Hello World](https://docs.github.com/en/get-started/quickstart/hello-world)
+* [YouTube - GitHub Tutorial For Beginners](https://www.youtube.com/watch?v=0fKg7e37bQE)
+* [YouTube - How To Fork A GitHub Repo and Submit A Pull Request](https://www.youtube.com/watch?v=G1I3HF4YWEw)
+* [YouTube - Markdown Crash Course](https://www.youtube.com/watch?v=HUBNt18RFbo)
+
+
+Don't hesitate to ask questions; every contributor started with a first PR. So... why not join our [large, growing](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=ebookfoundation/free-programming-books) community.
+
+<details align="center" markdown="1">
+<summary>Details about that growth</summary>
```

Comment:
unclear antecedent. How about "Click to see graph"

Reply:
> unclear antecedent. How about "Click to see graph"

Could be `<summary>Click to see graph details about that growth</summary>` or is too long?

Reply:
The antecedent of "that" is in a different paragraph; "graph details" is not really a thing.

How about "Cluck to see users vs. time graph."

Reply:
Resolved with rewording addresed at 8ec36ba883a81b832e4bee536e6a19b81a45242f